source/configure_project/frontmatter/index.html.md.erb

```diff
@@ -173,5 +183,14 @@ title: My beautiful page
 ---
 ```
 
+### description
```

Comment:
As above - to preserve alphabetical order, this would need to be the first h3 in this section.